freeipa/src/main/java/com/sequenceiq/freeipa/service/stack/NetworkService.java

```diff
@@ -57,11 +57,13 @@
                     // support for azure
                     String[] splittedNetworkId = cloudNetwork.getId().split("/");
                     String cloudNetworkId = splittedNetworkId[splittedNetworkId.length - 1];
-                    return networkId.equals(cloudNetworkId);
+                    // CB-9180 For GCP user can pass network name as id
```

Comment:
Actually, I have a different opinion here. It is interesting that both are valid points from each perspective.

A person who is reading that line of code, and click blame to read through my commit message. But a casual reader will miss that.

True story, `// support for azure` is actually what made me stop and think about what might be going wrong in that function for GCP!

Reply:
I have to say I would do it differently today, move that part to a method and give a meaningful name. If we add these comments to every part of the code then it will be outdated in a while etc. I still prefer avoiding comments as much as possible. Solving that issue and indicating with a comment was a mistake from my part.

Reply:
Ok, I have removed those comments to move forward.